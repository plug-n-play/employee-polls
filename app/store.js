import { configureStore } from "@reduxjs/toolkit";

import user from "@/features/user/userSlice";
import questions from "@/features/questions/questionsSlice";
import leaderboard from "@/features/leaderboard/leaderboardSlice";

export function makeStore() {
  return configureStore({
    reducer: {
      user,
      questions,
      leaderboard,
    }
  });
}

const store = makeStore();

export default store;
