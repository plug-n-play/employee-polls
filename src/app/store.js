import { configureStore } from "@reduxjs/toolkit";

import user from "@/features/user/userSlice";
import questions from "@/features/questions/questionsSlice";

export function makeStore() {
  return configureStore({
    reducer: {
      user,
      questions,
    }
  });
}

const store = makeStore();

export default store;
