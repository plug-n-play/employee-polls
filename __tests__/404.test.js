import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import { makeStore } from '@/app/store';
import PageNotFound from '@/pages/404';

test('renders 404 Page', () => {
  const store = makeStore();
  const { getByText } = render(
    <Provider store={store}>
      <PageNotFound />
    </Provider>
  );

  expect(getByText(/404: Page not found/i)).toBeInTheDocument();
  expect(render).toMatchSnapshot();
});