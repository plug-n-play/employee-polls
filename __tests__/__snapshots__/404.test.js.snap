// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 404 Page 1`] = `[Function]`;
